osbs/conf.py

```diff
@@ -548,7 +548,7 @@ def get_platform_node_selector(self, platform):
         if platform:
             nodeselector_str = self._get_value("node_selector." + platform, self.conf_section,
                                                "node_selector." + platform)
-            if nodeselector_str:
+            if nodeselector_str and '=' in nodeselector_str:
```

Comment:
can you give me the key, value pair you want here?
'node_selector.x86_64': 'amd64' will fail with this implementation, because it doesn't have an '=' in it. So I think I have fixed your case but I don't understand what you're asking for.